generators/server/templates/src/main/java/package/config/_LoggingConfiguration.java

```diff
@@ -48,13 +48,16 @@
 import org.springframework.cloud.consul.serviceregistry.ConsulRegistration;
 <%_ } _%>
 import org.springframework.context.annotation.Configuration;
+import org.springframework.cloud.context.config.annotation.RefreshScope;
```

Comment:
Please wrap this import statement with an "if serviceDiscovery enabled" template conditional so that it doesn't pollute the imports for application types who don't have service discovery.

Reply:
Hi @PierreBesson , thanks for the comment. i am not that familiar this the generator script.  Try my best to slove this. and you a welcom to edit it too.